### DISCUSSION TITLE
Qual a diferença entre JavaScript e TypeScripty?

### DISCUSSION THREAD

**Discussion by WellintonRafael** [WellintonRafael/pet-api#20] [Q&A] [answered]
Alguém sabe dizer quais as principais diferenças entre JS e TS?

**Comment by ghost** [comment, marked as answer]
JavaScript é uma linguagem de programação amplamente usada, interpretada e de tipagem dinâmica. TypeScript, por outro lado, é uma extensão de JavaScript que adiciona tipagem estática, compilação e outras funcionalidades avançadas. Enquanto JavaScript é mais flexível e rápido para escrever, TypeScript oferece verificação de tipo em tempo de compilação, ideal para projetos grandes e complexos. Em resumo, JavaScript é mais simples e direto, enquanto TypeScript é mais poderoso e robusto.
